tests/conftest.py

```diff
@@ -26,7 +26,7 @@
 ### TEST CONSTANTS ###
 
 SAMPLE_STRINGS_LIMIT = 256
-QUBITS_TO_BENCHMARK = [2, 4, 6, 9]
+QUBITS_TO_BENCHMARK = [2, 8]
```

Comment:
is this intentional? it's changing number of qubits we are using across all of our benchmarks

Reply:
agreed, good point